### PR TITLE
ENH: Add a script to update the ITK tag in Python `setup.py` files

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add a script to update the ITK tag in Python `setup.py` files.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
A few comments:
- This will surely need further testing.
- `jq` must be available.
- Assuming that the requirement filed is not empty.
- Assuming that no changes are made to the ``=` or `>=` symbol (although we will want to change to the latter I guess).
- Assuming all tags have the _v_ prefix.
- We may need to ask for the username so that the user is authenticated in the `curl` command. I think there is a limit in the anonymous gets.
- I guess generating a branch and automatically committing/pushing as @hjmjohnson has done in some scripts is also doable, but this is a first version.

File: Utilities/Maintenance/UpdateITKGitTagPythonSetup.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  echo "  param1 Setup.py file path"
+  exit 1
+}
+
+# Display usage
+if [ ${#@} != 1 ]; then
+  usage
+fi
+
+filename=$1
```
Comment (dzenanz) [inline Utilities/Maintenance/UpdateITKGitTagPythonSetup.sh]: I guess that same remark from remote update applies here regarding relative path to `Setup.py` file.

Comment (jhlegarreta) [reply Utilities/Maintenance/UpdateITKGitTagPythonSetup.sh]: Thanks @dzenanz . Answering in #641.

**Comment by jhlegarreta** [comment]
Superseded by #641 for the reasons stated in #634.

**Pull request closed without merging**